src/cmd-fetch

```diff
@@ -96,6 +96,11 @@ if [ $# -ne 0 ]; then
     fatal "ERROR: Too many arguments"
 fi
 
+if [ -n "${COSA_BUILD_WITH_BUILDAH:-}" ]; then
+    # shellcheck disable=SC2016
+    fatal 'ERROR: fetch is not useful when using $COSA_BUILD_WITH_BUILDAH'
```

Comment:
Hmm, I was thinking instead we would just no-op.

That'll avoid having to conditionalize it in the pipeline.

Reply:
changed it to a no-op.